be/src/Infrastructure/Localization/Foundation.Localization/Resources/GlobalResources.de.resx

```diff
@@ -166,7 +166,7 @@
     <value>Link kopieren</value>
   </data>
   <data name="Copyright" xml:space="preserve">
-    <value>Copyright © [year]</value>
+    <value>© [year] | ELTAP Spółka z ograniczoną odpowiedzialnością Sp.k. Alle Rechte vorbehalten.</value>
```

Comment:
**suggestion:** lets' move these texts to Content.API because when I deploy it to other tenant the text is going to be invalid